### DISCUSSION TITLE
email template

### DISCUSSION THREAD

**Discussion by Eurovisionopticos** [PrestaShop/PrestaShop#36447] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I just replaced a placeholder in an email template with two words. Unfortunately, this change has caused the template to look disorganized.

Is there a way to reset the original version of the email template?
![email to customer now](https://github.com/PrestaShop/PrestaShop/assets/174035712/c7fb0827-e1d3-4b4a-80ff-5776215f784b)

### Expected behavior

![email to customer before](https://github.com/PrestaShop/PrestaShop/assets/174035712/b40d74d3-a050-45c5-9b92-124a57406a1c)

### Steps to reproduce

![email to customer now](https://github.com/PrestaShop/PrestaShop/assets/174035712/c4613fd8-8a0b-43c8-a63a-0f5c08db8dd1)

### PrestaShop version(s) where the bug happened

8.1.4

### PHP version(s) where the bug happened

8.1.4

### Your company or customer's name goes here (if applicable).

Eurovision Opticos

**Comment by kpodemski** [comment]
I transferred your question to Discussions since it is not a bug.

You could replace file with the original one.

**Reply by Eurovisionopticos** [reply]
Please indicate where I can find the original file and how to replace it with the current one.
